CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ### Features
 - Consider safe area when displaying interstitial Ads for Standalone, In-House, MoPub AppBidding and
   mediation adapters. DFP and Custom AppBidding are still always fullscreen.
+--------------------------------------------------------------------------------
+## Version 3.9.0
+
+Improvements and fixes
```

Comment:
Could you explain a little bit more what this means ?

Reply:
No I can't in fact 🍭